### PR TITLE
a few minor updates to the analysis scripts

### PR DESCRIPTION
**Pull request by tomsilver**

### CODE REVIEW

File: analysis/skeleton_score_analysis.py [hunk 1]
```diff
@@ -252,7 +253,10 @@ def get_default_result(_skeleton_idx: int) -> float:
 
 def _create_predicate_labels(
         predicate_set_order: Tuple[FrozenSet[Predicate], ...]) -> List[str]:
-    labels = [", ".join(p.name for p in predicate_set_order[0])]
+    if len(predicate_set_order[0]) > 5:
```
Comment (ronuchit) [inline analysis/skeleton_score_analysis.py:256]: is this a hack?

File: analysis/create_latex_tables.py [hunk 2]
```diff
@@ -188,8 +226,8 @@ def _main() -> None:
                     inner_label_to_best_mean_std[inner_label] = (mean, std)
         for (outer_label, inner_label), (mean, _,
                                          _) in entry_to_mean_std_size.items():
-            # Special case: exclude oracle
-            if outer_label == "Oracle":
+            # Special case: exclude Handwritten
+            if outer_label == "Handwritten":
```
Comment (ronuchit) [inline analysis/create_latex_tables.py:230]: ah, thank you for remembering to do this!

**Review by ronuchit — APPROVED** [review]
thank you!!

File: analysis/skeleton_score_analysis.py (hunk 1 above)
Comment (tomsilver) [reply analysis/skeleton_score_analysis.py:256]: yeah just for making the axis labels not crazy

**Pull request merged**